### PR TITLE
SOSSE-8946: update white list urls

### PR DESCRIPTION
**Pull request by CharlesWang1**
https://jira.labs.quest.com/browse/SOSSE-8946

### CODE REVIEW

File: pages/SpotlightEnterprise/enterprise_upgrade.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ To upgrade from an earlier version of Spotlight on SQL Server than 10.0, upgrade
 
 To upgrade from Spotlight on SQL Server 10.5.0, first upgrade to 10.5.2 and then upgrade to Spotlight Enterprise.
 
+### Spotlight Enterprise 13.2 and Internet Access
+On upgrade to Spotlight Enterprise 13.2, If Windows Firewall is set up to block traffic by default then in Windows Firewall | Advanced Settings add new inbound and outbound rules to enable the following URLs on TCP port 443:
```
Comment (RGriffin-Quest) [inline pages/SpotlightEnterprise/enterprise_upgrade.md:45]: I think an inbound rule is not required.

File: pages/SpotlightEnterprise/enterprise_releasenotes_systemrequirements.md [hunk 2]
```diff
@@ -260,6 +233,23 @@ It is highly recommended that regular maintenance is performed on the Spotlight
 ## Spotlight Cloud
 Cloud Connect is compatible with SoSSE versions 13.2 and higher. This is effective from the 1st of June 2020.
 
+#### Internet Access
+Required by Spotlight Cloud. Required to monitor your Spotlight connections from the Spotlight Cloud web site or a mobile device.
+
+If Windows Firewall is set up to block traffic by default then in Windows Firewall | Advanced Settings add new inbound and outbound rules to enable the following URLs on TCP port 443:
```
Comment (RGriffin-Quest) [inline pages/SpotlightEnterprise/enterprise_releasenotes_systemrequirements.md:239]: I think inbound rules are not required. The DS doesn't accept connection from these web sites.

File: pages/SpotlightEnterprise/enterprise_upgrade.md (hunk 1 above)
Comment (SimonDu123) [reply pages/SpotlightEnterprise/enterprise_upgrade.md]: @RGriffin-Quest when we acknowledge or snoozed alarm from the website, does DS receive the event from the website?

**Comment by CharlesWang1** [comment]
@RGriffin-Quest updated please review again

**Pull request merged**